### DISCUSSION TITLE
Welcome to wtfjs Discussions!

### DISCUSSION THREAD

**Discussion by denysdovhan** [denysdovhan/wtfjs#183] [Discussions]
## 👋 Welcome!

We’re using Discussions as a place to connect with other people interested in exploring JavaScript's depths. We hope that you:

* Ask questions you’re wondering about.
* Answer questions you can answer.
* Are kind, respectful, and open-minded. Remember that this is a community we build together 💪.

**Comment by Gutiee001** [comment]
Hello Sir, how can I download the source code?

**Reply by denysdovhan** [reply]
You can clone, fork or download this repo as long as you're compliant with this license: https://github.com/denysdovhan/wtfjs/blob/master/LICENSE
